.github/workflows/tests.yaml

```diff
@@ -4,17 +4,24 @@ on: [pull_request, create]
 jobs:
   build:
     name: Tests
-    runs-on: ubuntu-22.04
+    runs-on: ${{ matrix.runner }}
     strategy:
       fail-fast: false
+      matrix:
+        include:
+          - runner: ubuntu-22.04
+            coreboot-tests: true
+          - runner: focal-arm64
+            coreboot-tests: false
     steps:
       - name: Code checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
       - name: Run unit tests
         run: scripts/dev_cli.sh tests --unit
       - name: Run integration tests
         run: scripts/dev_cli.sh tests --integration
-      - name: Run integration tests
-        run: scripts/dev_cli.sh tests --integration-coreboot
\ No newline at end of file
+      - if: ${{ matrix.coreboot-tests }}
+        name: Run coreboot integration tests
```

Comment:
Thanks for fixing this - I saw this the other day when I was reviewing the output.